internal/process/update/update_runtime_step_test.go

```diff
@@ -131,12 +131,56 @@ func TestUpdateRuntimeStep_RunUpdateEmptyOIDCConfigWithOIDCObject(t *testing.T)
 	assert.Equal(t, expectedOIDCConfig, (*gotRuntime.Spec.Shoot.Kubernetes.KubeAPIServer.AdditionalOidcConfig)[0])
 }
 
+func TestUpdateRuntimeStep_RunUpdateRemoveJWKSConfig(t *testing.T) {
```

Comment:
Sugested test cases:

1. Updating existing JWKS with a new value
2. Updating non-existing JWKS with a value

Could be done in one pass (one test case) using list. One OIDC item with JWKS, one without and we update both OIDC items, we may add third item with JWKS set and we leave it untouched (empty param).